src/ConfigResource.php

```diff
@@ -305,14 +356,16 @@ public function createNestedKeyValuePair(&$patchValues, $key, $value)
     protected function extractAndSet(array $keys, $value, &$array)
     {
         $key = array_shift($keys);
-        if ($keys) {
+        if (count($keys)) {
```

Comment:
Why this change?

Reply:
I think these are not the same rows as I don't recall changing this.